tests/unittests/test_timeout.py

```diff
@@ -0,0 +1,222 @@
+from datetime import datetime
+import socket
+import unittest
+from unittest import mock
+from tap_sftp import client, discover, sync
+
+class TestTimeoutValue(unittest.TestCase):
+    """
+        Test case to verify the timeout value is set as expected
+    """
+
+    def test_timeout_value_not_passed_in_config(self):
```

Comment:
Please add comments for each function, what it is testing.

Reply:
Added comment for each test case function.